packages/web-ui/src/lab/Link/Link.docs.mdx

```diff
@@ -0,0 +1,128 @@
+import { Meta, Canvas, ArgTypes, Description } from '@storybook/blocks';
+import * as Stories from './Link.stories';
+
+<Meta of={Stories} />
+
+# Link
+
+<Description />
+
+<Canvas of={Stories.SimpleExample} />
+
+## Usage
+
+This `Link` component is part of the lab submodule, a way for us to make new
+components available prior to an official major release. This helps us bed in
+new components before introducing breaking changes, and to avoid blocking the
+availability of these components for designers.
+
+There are 2 options for importing this component.
+
+1. Import directly from the `lab` directory.
+
+```tsx
+import { Box } from '@utilitywarehouse/web-ui';
+import { Link } from '@utilitywarehouse/web-ui/dist/lab';
+
+[...]
+
+<Box>
+  <Link>Link</Link>
+</Box>
+```
+
+2. Import `Lab` directly from Web UI.
+
+```tsx
+import { Box, Lab } from '@utilitywarehouse/web-ui';
+
+[...]
+
+<Box>
+  <Lab.Link>Link</Lab.Link>
+</Box>
+// or
+const { Link } = Lab
+<Box>
+  <Link>Link</Link>
+</Box>
+```
+
+## Size
+
+The size prop controls the size of the `Link`. This is a responsive prop and
```

Comment:
It could be good clarify that you can also override this, because sometimes designers use a Large link on smaller screen size to have a larger target area

Reply:
So, the size itself is not responsive, it will only change at certain breakpoints if you tell it to.

Reply:
by responsive prop, I mean that it accepts an object declaring what size the component should be at specific screen sizes.